docs/data/material/components/init-color-scheme-script/init-color-scheme-script.md

```diff
@@ -0,0 +1,140 @@
+---
+productId: material-ui
+title: InitColorSchemeScript component
+components: InitColorSchemeScript
+githubSource: packages/mui-material/src/InitColorSchemeScript
+---
+
+# InitColorSchemeScript
+
+<p class="description">InitColorSchemeScript component removes the dark mode flicker for server-side rendering application.</p>
+
+## Introduction
+
+`InitColorSchemeScript` component is used to remove the dark mode flicker for server-side rendering (SSR) applications.
+It is a script that runs before React to attach an attribute based on user preference.
```

Comment:
```suggestion
This script that runs before React to attach an attribute based on the user preference so that the correct color mode is applied on first render.
```